cmds/core/gpgv/gpgv.go

```diff
@@ -113,10 +83,45 @@ func verifyDetachedSignature(key *packet.PublicKey, contentf, sigf io.Reader) er
 	switch sig := p.(type) {
 	case *packet.Signature:
 		err = key.VerifySignature(h, sig)
-	case *packet.SignatureV3:
-		err = key.VerifySignatureV3(h, sig)
 	default:
-		panic("unreachable")
+		return fmt.Errorf("unknown signature")
 	}
 	return err
 }
+
+func runGPGV(w io.Writer, verbose bool, keyfile, sigfile, datafile string) error {
+	if keyfile == "" || sigfile == "" || datafile == "" {
+		return errUsage
+	}
+
+	keyf, err := os.Open(keyfile)
```

Comment:
I notice there are no closes. Here is one case where we could use ReadFile, namely for the key and the sigfile, as these are generally small IIUC. Then add a deferred close of the datafile, I guess?